packages/vaadin-lumo-styles/.gitignore

```diff
@@ -0,0 +1 @@
+auto-complete.css
```

Comment:
Would it make sense to add this file to the root level `.gitignore`?

Reply:
Right, I didn't notice that we don't have any nested ignore files. Moved to root `.gitignore`.